app/Models/RockTheVoteReport.php

```diff
@@ -52,15 +53,19 @@ class RockTheVoteReport extends Model
     ];
 
     /**
-     * Logs a Rock The Vote Report that we've created via API request.
+     * Creates a Rock The Vote Report via API request and saves to storage.
      *
-     * @param object $response
      * @param datetime $since
      * @param datetime $before
      * @return RockTheVoteReport
      */
-    public static function createFromApiResponse($response, $since, $before)
+    public static function createViaApi($since = null, $before = null)
```

Comment:
We won't always be using `$since` and `$before`?

Reply:
For our purposes... we do always pass these parameters currently, but the API doesn't require them (if we wanted to generate a report for all VR's for example). This felt cleaner than passing `null` as parameter values in the tests (hence this change)

Reply:
Oh nice, sounds good!